src/test/java/rx/SingleTest.java

```diff
@@ -38,10 +38,13 @@
 import rx.functions.Action1;
 import rx.functions.Func1;
 import rx.functions.Func2;
+import rx.observables.BlockingSingle;
 import rx.observers.TestSubscriber;
 import rx.schedulers.Schedulers;
 import rx.subscriptions.Subscriptions;
 
+import static org.junit.Assert.*;
```

Comment:
Please don't use wildcard imports, they hide changes of used APIs.

Looks like you use IntelliJ IDEA — Settings -> Code Style -> Java -> Imports -> "Class count to use import *" and "Names count to use static import with *" just set some huge value.